extension/opampextension/config.go

```diff
@@ -148,6 +150,17 @@ func (s OpAMPServer) GetEndpoint() string {
 	return ""
 }
 
+func (s OpAMPServer) GetAuthExtensionID() component.ID {
```

Comment:
can you unexport this method?

Reply:
I can, that would be inconsistent with every other getter method on this config though. Should I make an issue to unexport all of them? Seems like it would be a good idea.

Reply:
I'm okay doing this in a follow-up.